bitnami/postgresql/Chart.yaml

```diff
@@ -28,4 +28,4 @@ name: postgresql
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/postgresql
   - https://www.postgresql.org/
-version: 11.9.3
+version: 11.9.4
```

Comment:
Please rebase your branch and set a new chart version: https://github.com/bitnami/charts/blob/master/bitnami/postgresql/Chart.yaml

Reply:
The latest version is now 11.9.5